src/contexts/usePortkey/provider.tsx

```diff
@@ -11,7 +11,9 @@ import {
 } from '@portkey/provider-types';
 import detectProvider from '@portkey/detect-provider';
 import { evokePortkey } from '@portkey/onboarding';
-import { PortkeyNameVersion, PORTKEY_APP_DOWNLOAD_PAGE } from './constants';
+import { PortkeyNameVersion, PortkeyVersion, PORTKEY_APP_DOWNLOAD_PAGE } from './constants';
+import { isMobileDevices } from 'utils/isMobile';
+import { isPortkey } from 'utils/portkey';
```

Comment:
this function will have to be renamed to better suit its purpose for readability.

Reply:
Noted down as refactor of ebridge code (Separate sprint)